client/my-sites/themes/themes-selection.jsx

```diff
@@ -218,7 +224,7 @@ export const ConnectedThemesSelection = connect(
 		const query = {
 			search,
 			page,
-			tier: config.isEnabled( 'themes/premium' ) ? tier : 'free',
+			tier: isSupportPremiumThemes ? tier : 'free',
```

Comment:
Oh, maybe this should be hiding the themes. Let me check the value of `isSupportPremiumThemes` on my Jetpack site.

Reply:
It's false, as expected.

Reply:
Yes, this part was supposed to be hiding it. :thinking: